lib/workflows.js

```diff
@@ -5,15 +5,99 @@
  *
  */
 'use strict';
-
+const database = require('./database');
+const utils = require('./utils');
 const load = require('./workflows/load');
 const data = require('./workflows/data');
 
-module.exports = data;
 
-/**
- * Get all Workflows
+/*
+ * @typedef AuditEntry
+ * @type object
+ *
+ * @property {string} comment approver's comment
+ * @property {string} action  action taken.
+ * @property {number} step  current step in workflow
+ * @property {number} author  author of audit entry
+ * @property {FormattedDate} - The individually formatted date components
+ */
+
+/*
+ * An audit entry
+ *
+ * @param {string} comment approver's comment
+ * @param {string} action  action taken.
+ * @param {number} step  current step in workflow
+ * @param {object} req - HTTP Request
+ *
+ * @returns {AuditEntry} audit entry
+ */
+const entry = (comment, action, step, req) => {
+  const data = {
+    comment,
+    action,
+    step,
+    author: req.user.id,
```

Comment:
Can we change this from `user.id` to either be an object that contains the `id` and the `email`, or just the `email` of the user? If a user is deleted from the system, the `id` is meaningless for the audit trail